### DISCUSSION TITLE
Why so many star.p.backup?

### DISCUSSION THREAD

**Discussion by Solarunit** [Taiko2k/Tauon#740] [Q&A] [answered]
I wonder why there are so many `star.p.backup` files in the `TauonMusicBox` folder?

![](https://i.imgur.com/0C5Ymdo.png)

I assume that tauon makes backup after the each big update? But why only star.p? :thinking:

**Comment by Taiko2k** [comment, marked as answer]
It leaves one for every month of the year, so in case of corruption, the most recent backup should be at most 1 month old. Haven't come up with a better system.

It's backed up more because it's contains the playtimes and ratings, which aren't so easily replaceable.

**Reply by Solarunit** [reply]
gotcha, thanks!
